ekm/signer_key_manager_test.go

```diff
@@ -36,15 +35,22 @@ const (
 	pk2Str = "8796fafa576051372030a75c41caafea149e4368aebaca21c9f90d9974b3973d5cee7d7874e4ec9ec59fb2c8945b3e01"
 )
 
-func testKeyManager(t *testing.T) spectypes.KeyManager {
+func testKeyManager(t *testing.T, network *networkconfig.NetworkConfig) spectypes.KeyManager {
```

Comment:
This is never used, network is always `nil`

Reply:
there is an option to pass the network in order to simulate the current slot.
will be relevant for future slashing protection tests.